commonlibktx/src/main/java/com/crisolutions/commonlibraryktx/Extensions.kt

```diff
@@ -0,0 +1,17 @@
+package com.crisolutions.commonlibraryktx
+
+/**
+ * Not sure what other class to place these in.
+ * */
+
+
+// Continues the chain only if block is true.
+
+//
+fun <T> T.filterIf(block: (T) -> Boolean): T? =
+        if (block.invoke(this)) {
```

Comment:
I think you can remove the curly braces and make this a single line

Reply:
Updated.